### PR TITLE
Finish desktop version

### PR DESCRIPTION
**Pull request by benshidanny11**
- Added contact form for the desktop version
- Made styling for the desktop version.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @benshidanny11, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: contactme.css [hunk 1]
```diff
@@ -89,43 +85,41 @@
...
- margin-top: 3.5rem;
- display: flex;
- justify-content: center;
+.imgcontainer-contact {
+  margin-top: 3.5rem;
+  display: flex;
+  justify-content: center;
 }
 
 @media only screen and (min-width: 720px) {
```
Comment (mke2111) [inline contactme.css]: - [ ] The value for min-width property should be in range of 992px - 1100px. These are the accepted desktop screen breakpoints. Change your value. What you are using lies under the tablet screen breakpoints.

Comment (mke2111) [inline contactme.css]: - [ ] The about section has a few details that are not matched.
Align the left column with the right column from the top.
Screenshot from 2021-08-19 12-52-41

File: contactme.css [hunk 2]
```diff
@@ -89,43 +85,40 @@
...
+.imgcontainer-contact {
+  margin-top: 3.5rem;
+  display: flex;
+  justify-content: center;
 }
 
-@media only screen and (min-width: 720px) {
-
-  .contact-container-flex{
+@media only screen and (min-width: 992px) {
```
Comment (mke2111) [inline contactme.css:94]: - [ ] - [ ] Add some padding/margin between these two columns. Do this for each of the projects.
Screenshot from 2021-08-19 12-54-31

**Review by mke2111 — COMMENTED** [review]
.

File: contactme.css (hunk 1 above)
Comment (benshidanny11) [reply contactme.css]: Thanks, But am not getting what you mean! Please clarify your comment.

**Review by AshakaE — APPROVED** [review]
### Hi @benshidanny11  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**